rules-reviewed/quarkus/springboot/springboot-metrics-to-quarkus.windup.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="springboot-metrics-to-quarkus" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset gives hints to migrate from SpringBoot Metrics to Quarkus Microprofile Metrics extension
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-xml,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="springboot"/>
+        <targetTechnology id="quarkus"/>
+    </metadata>
+    <rules>
+        <rule id="springboot-metrics-to-quarkus-0100">
+            <when>
+                <dependency groupId="io.micrometer" artifactId="micrometer-core"/>
+            </when>
+            <perform>
+                <hint title="Replace the Micrometer dependency with Quarkus 'metrics' extension" effort="5" category-id="mandatory">
```

Comment:
worth adding `MicroProfile` like mentioned in all the titles of the other rules.
```suggestion
                <hint title="Replace the Micrometer dependency with Quarkus MicroProfile 'metrics' extension" effort="5" category-id="mandatory">
```

Reply:
+1